apps/masterbots.ai/components/routes/profile/user-card.tsx

```
@@ -0,0 +1,199 @@
+                src={user?.profilePicture ? user.profilePicture : ''}
+                alt="Default Avatar"
+                height={128}
+                width={128}
+              />
+            </div>
+            <div className='absolute bottom-4 right-0 p-2 rounded-full  dark:bg-[#83E56A] bg-[#BE17E8]'>
+                <ImagePlus className="w-3 h-3  rounded-full dark:text-black text-white font-bold" />
+            </div>
+          </div>
+          {!isOwner && (
+          <button className="px-4 py-1 text-sm text-white  rounded-md bg-[#BE17E8] hover:bg-[#BE17E8] dark:bg-[#83E56A] dark:hover:bg-[#83E56A] dark:text-black transition-colors">
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)